src/features/movies/MovieDetails/Backdrop/index.js

```diff
@@ -0,0 +1,28 @@
+import { PosterWrapper, PosterDisplay, MainInfoWrapper, MainInfoTitle } from "./styled";
+import { RatingWrapper, StarIcon, Rate, Votes } from "../../../../common/Details";
+import { apiBackdrop } from "../../../../apiData";
+
+const Backdrop = movie => {
+  return (
+    <PosterDisplay>
+      <PosterWrapper style={{ backgroundImage: `url(${apiBackdrop}${movie.movie.backdrop_path})` }}>
```

Comment:
maybe we should deleted style and add background-image to styles components with url + props src?

Reply:
Fixed - finally found a way to pass something as props in styled components. It is working as before, but looks a lot better in code.

Reply:
Yes, now looks better without style - great job!